app/controllers/hyrax/my/collections_controller.rb

```diff
@@ -24,7 +27,7 @@ def self.configure_facets
       def index
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.admin.sidebar.collections'), hyrax.dashboard_collections_path
+        add_breadcrumb t(:'hyrax.admin.sidebar.collections'), hyrax.my_collections_path
```

Comment:
We don't want to send folks to my_collections - this only shows collections you own instead of all collections - that's not really useful for admin users.

We want to send folks to the view that lists all collections by default.

Reply:
Probably check in with @norm0  on this one.

Reply:
oh, yeah, that was a fat finger on my part